formats/developers/6d-7d-and-8d-storage.txt

```diff
@@ -206,29 +205,41 @@ mosaic as above or to overlap e.g.
 
     E   <Plane TheC="0" TheT="12" TheZ="0" PositionX="80" PositionY="110" PositionZ="0.1"/>
 
-Sample Files
-^^^^^^^^^^^^
+Sample files
+------------
 
+Sample files are available from the
+:image_downloads:`Modulo <OME-TIFF/2015-01/modulo/>` folder of the image downloads resource.
```

Comment:
let's discuss, I agree with @mtbc that we might want to define the schema in `conf.py` esp. as we don't bump it often. In that case it might be possible to define specific OME-TIFF and OME-XML extlinks pointing to the samples for the latest version of the schema, e.g.

```
 'ometiff_downloads' : (downloads_root + '/images/OME-TIFF/' + release + '/%s', ''),
```